00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.02 Design Decisions/hub.yaml proposal for themes.md

```diff
@@ -0,0 +1,303 @@
+
+```yaml
+theme:
+    plugin_support: [] # IDs of the plugins that have been styled by this theme (adding them here would ignore the plugin IDs on the obsidian.css file)
+```
+
+### Screenshots
+
+If you want to add additional screenshots, or don't want to add features but prefer to just add screenshots of your theme, you can use the `screenshot` key:
+
+```yaml
+theme:
+    screenshots: # Path to other screenshots from the root of the repo
+        - file: 
+          caption:
+```
+
+## Template
+
+Here is a full template with all the options being proposed:
+
+```yaml
+docs: # Link to the theme documentation
+hub_content: [], # List of markdown documents to be included as is to the hub relative to the root of the repo.
+authors: # More than one author is possible, start a new line with - 
+    - github: # GitHub user names of the people who created the plugin or theme
+      websites: [] # Links to personal websites
+      discord:
+      twitter:
+      youtube:
+      sponsorships:
+          buymeacoffee: # Add username to kofi or buy me a coffee here
+          paypal: 
+          github:
+          patreon:
+          others: [] # List of formatted markdown strings
+maintainers: [] # GitHub user names of those maintaining the plugin (leave empty if it's the same as author)
+
+theme:
+    features: 
+        mobile:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        publish: true
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        gutter:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        print:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        callouts:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        checklists:
```

Comment:
these are more commonly known as custom checkboxes

Reply:
Should be fixed in 524cc40